rendering/markdown.go

```diff
@@ -41,6 +41,21 @@ type highlightHTMLRenderer struct {
 	blackfriday.Renderer
 }
 
+// ListItem overrides the default ListItem render and adds support for GH-style 
+// checkboxes on service side. For the contents of the list item beyond the
+// checkbox prefix, the default Html.ListItem is called.
+func (h highlightHTMLRenderer) ListItem(out *bytes.Buffer, text []byte, flags int) {
+	switch {
+		case bytes.HasPrefix(text, []byte("[] ")):
```

Comment:
I believe GH might have a reason not to support `[]` if there's no space between the backets.

Reply:
Is there an antipattern?

Reply:
I don't know I just double checked.

Reply:
I think we should support this. I'll run into this issue all the time when doing GH work - unless there is a specified antipattern.